bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutation.java

```diff
@@ -71,7 +74,7 @@ public BulkMutation(String tableName) {
   }
 
   public boolean isFull() {
-    return size >= MAX_SIZE || futures.size() >= MAX_COUNT;
+    return size >= maxRpcSize || futures.size() >= maxRpcCount;
```

Comment:
I'm renaming this to reachedRequestedMax() which is hopefully more indicative of the intent of the method.  It's more like the user sets a threshold that can be ignored by the user, if they want to.  I'd rather not copy the logic to two places if I can avoid it.